### ISSUE TITLE
TOC missing in PDF

### ISSUE THREAD

**Issue by SuibianP** [SuibianP/pe#14]
While the hyperlinks are there, the TOC outline supported in Adobe PDF format is missing. This causes difficulties in navigation and accessibility (e.g. screen reader).

![image.png](https://raw.githubusercontent.com/SuibianP/pe/main/files/85d6ea17-12da-4ed2-88e3-a0e1dd820d86.png)
